r274105 | amodra | 2019-08-05 01:52:33 +0000

PR91349, powerpc64*-*-freebsd* defines _GNU_SOURCE

rev 266496 (git ab6b1bb456) undefined some macros in rs6000/freebsd.h
but missed doing the same in rs6000/freebsd64.h.

        PR target/91349
        * config/rs6000/freebsd64.h (CPLUSPLUS_CPP_SPEC),
        (LINK_GCC_C_SEQUENCE_SPEC): Undef.

--- UTC
Index: gcc/config/rs6000/freebsd64.h
===================================================================
--- gcc/config/rs6000/freebsd64.h	(revision 274104)
+++ gcc/config/rs6000/freebsd64.h	(revision 274105)
@@ -17,6 +17,10 @@
    along with GCC; see the file COPYING3.  If not see
    <http://www.gnu.org/licenses/>.  */
 
+/* Undef gnu-user.h macros we don't want.  */
+#undef CPLUSPLUS_CPP_SPEC
+#undef LINK_GCC_C_SEQUENCE_SPEC
+
 /* Override the defaults, which exist to force the proper definition.  */
 
 #ifdef IN_LIBGCC2
